### ISSUE TITLE
Tweak autoClosingPairs to make JavaScript docblock insertion less intrusive

### ISSUE THREAD

**Issue by kAlvaro** [Microsoft/vscode#51131]
As soon as you type `/**` you immediately get `/**𝖨 */` (where 𝖨 is the cursor). If you didn't intend to start a docblock you need to remove the extraneous `*/`. I'd appreciate two enhancements:

- Expand to docblock only after `/**<space>` or `/**<enter>`.
- Disable the overall feature if you're already inside a comment and maybe if the line already has code before the caret. (Other invalid contexts seem to be detected nicely.)

This is a follow-up to #51047.

**Comment by mjbvz** [comment]
Duplicate of #1966
